packages/web-app-files/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "AGPL-3.0",
   "dependencies": {
     "copy-to-clipboard": "^3.3.1",
-    "semver": "^6.1.0",
+    "semver": "^7.3.5",
```

Comment:
Need to revert this later because of https://github.com/owncloud/web/pull/5832

Reply:
@kulmann could you open an issue outlining why semver >`v6` breaks the archiver? Would be nice to get it streamlined across our repos:)

Reply:
Would need to look into that again. As far as I remember the v6 was required by another dependency... need to check again which one it was.

Reply:
Maybe ironically the SDK? :D

Reply:
SDK is on `v7.3.5` now on current master, so might have been the case. Let's go for https://github.com/owncloud/owncloud-sdk/pull/979 after this PR is merged so we can save shipping axios&semver twice with our dependencies :)